qt@4.rb

```diff
@@ -134,11 +142,10 @@ def caveats; <<-EOS.undent
     We agreed to the Qt opensource license for you.
     If this is unacceptable you should uninstall.
 
-    Qt Designer no longer picks up changes to the QT_PLUGIN_PATH environment
-    variable as it was tweaked to search for plug-ins provided by formulae in
-      #{HOMEBREW_PREFIX}/lib/qt4/plugins
-
     Phonon is not supported on macOS Sierra or with Xcode 8.
+    
+    Webkit is no longer included -- please use `brew install qt-webkit@2.3` to
```

Comment:
People will blindly run this and ignore the messaging in that formula that tells them not to.